slam/feature_extractor/superpoint/weights/README.rst

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
this file seems unnecessary. add weights link to top of model.py and remove this file?

Reply:
i just create that file only for showing folder structure.  without that file weights folder will be empty. gitignore  ignore the empty folders

Reply:
That's allright. we don't need an empty dir. Can auto create the dir at the time of downloading weights.

Reply:
You can delete the file